finish/action.yml

```diff
@@ -32,8 +32,8 @@ runs:
       shell: bash
       if: inputs.include-preview
       run: |
-        if [[ -n "${{ inputs.preview-url }}" ]]; then
-          echo "LS_PREVIEW_URL=${{ inputs.preview-url }}" >> $GITHUB_ENV
+        if [[ -n "${LS_PREVIEW_URL:-${{ inputs.preview-url }}}" ]]; then
+          echo "LS_PREVIEW_URL=${LS_PREVIEW_URL:-${{ inputs.preview-url }}}" >> $GITHUB_ENV
```

Comment:
preface: I'm not the bash expert

I'm pretty sure this does not check whether `LS_PREVIEW_URL` is already set in environment variables.
It's probably missing another check for `env.LS_PREVIEW_URL`

Reply:
The env variables itself is checked with the `test -n` flag.
The trick here the above format `${LS_PREVIEW_URL:-ANY_STRING}` bash variable expansion, so when `LS_PREVIEW_URL` is empty or not set bash automatically substitutetes it's value to the value after the `:-`.
This enforces the desired precedence where the env variable is higher than the input parameter without the need of if/elif/else conditions.